yaml/xyz/openbmc_project/Inventory/Item/Cable.interface.yaml

```diff
@@ -17,3 +17,64 @@ properties:
           in user interfaces but this field should not be used for programmatic
           interrogation of a cable beyond ignoring the default value of the
           empty string.
+    - name: CableStatus
+      type: enum[self.Status]
+      default: 'Unknown'
+      description: >
+          The status of the cable. The default status is unknown.
+
+enumerations:
+    - name: Status
+      description: >
+          Possible cable status
+      values:
+          - name: 'Inactive'
+            description: >
+                Cable is inactive.
+
+          - name: 'Running'
+            description: >
+                Cable is running.
+
+          - name: 'PoweredOff'
+            description: >
+                Cable is powered off.
+
+          - name: 'Unknown'
+            description: >
+                Cable status is unknown.
+
+associations:
+    - name: downstream_chassis
+      description: >
+          Objects that implement Cable can optionally implement the
+          downstream_chassis association to provide a link to the
+          chassis.
+      reverse_name: attached_cables
+      required_endpoint_interfaces:
+        - xyz.openbmc_project.Inventory.Item.Chassis
+
+    - name: upstream_chassis
+      description: >
+          Objects that implement Cable can optionally implement the
+          uptream_chassis association to provide a link to the chassis.
+      reverse_name: attached_cables
+        - xyz.openbmc_project.Inventory.Item.Chassis
+
+associations:
```

Comment:
we dont need two key right ? as all are associations ?

Reply:
done